pkg/frontend/nvme_controller.go

```diff
@@ -115,22 +106,31 @@ func (s *Server) CreateNvmeController(_ context.Context, in *pb.CreateNvmeContro
 		log.Printf("client provided the ID of a resource %v, ignoring the name field %v", in.NvmeControllerId, in.NvmeController.Name)
 		resourceID = in.NvmeControllerId
 	}
-	in.NvmeController.Name = server.ResourceIDToVolumeName(resourceID)
```

Comment:
I think Name should be full resource name and not just ID...

Reply:
resloved